source/_integrations/securitas_direct.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Securitas Direct
+description: Instructions on how to setup Securitas Direct home security system on Home Assistant.
+ha_category:
+  - Hub
+  - Alarm
+ha_release: 2020.12
+ha_iot_class: Cloud Polling
+ha_domain: securitas_direct
+---
+
+The `securitas_direct` integration allows users to manage their Securitas Direct installation
+There is currently support for the following device types within Home Assistant:
+- **Alarm Control Panel**: Reports on the current alarm status and can be used to arm and disarm the system.
+
+## Configuration
+
+To integrate Verisure with Home Assistant, add the following section to your `configuration.yaml` file:
+
+Securitas Direct can be integrated by adding the following `securitas_direct` section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+securitas_direct:
+  username: USERNAME
+  password: PASSWORD
+  installation: INSTALLATION_NUMBER
+  code: 1234
+  country: PT
+  lang: pt
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>